velox/exec/tests/HashJoinTest.cpp

```diff
@@ -200,6 +203,12 @@ using JoinResultsVerifier =
 
 class HashJoinBuilder {
  public:
+  enum SpillOption {
```

Comment:
Why we need this? If it is for test refactoring, let's keep it separate? Thanks!